### PR TITLE
Phase 6

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description
This PR adds Phase 6 of the Primer. It will only be ready when the chain is fixed.

### CODE REVIEW

File: phase-6/block-unblock.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+To block or unblock a user, you will need their address.  
+:::
+
+### Blocking a user
+To block a user, you have to use the following command:
+
+```shell
+desmoscli tx relationships block <User address> <Subspace> <(Optional) Reason> \
+  --from <your-key-name> --yes \
+  --chain-id moprheus-8000
```
Comment (kwunyeung) [inline phase-6/block-unblock.md:21]: Should this be a placeholder like other values?
```suggestion
  --chain-id <chain-id>
```

File: phase-6/block-unblock.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+This will return you the JSON representation of the transaction itself.
+
+### Unblocking a user
+To unblock a user, you have to use the following command:
+
+```shell
+desmoscli tx relationships unblock <User address> <Subspace> \
+  --from <your-key-name> --yes \
+  --chain-id moprheus-8000
```
Comment (kwunyeung) [inline phase-6/block-unblock.md:70]:
```suggestion
  --chain-id <chain-id>
```

File: phase-6/transfer-dtag.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+In the future that `.md` suffix will be removed, but for now you need to use it unless specified differently.  
+:::
+
+The `<New DTag>` will be the new DTag that is going to be associated to them once they gave you their current DTag.  
+The `<Sender address>` must be the same address of the user that has started the DTag transferring. 
+
+Here's an example of such command: 
+
+```shell
+desmoscli tx profiles accept-dtag-transfer.md \
```
Comment (kwunyeung) [inline phase-6/transfer-dtag.md:103]: Would this https://github.com/desmos-labs/desmos/pull/282 be tagged and refactor the correct command here?

Comment (RiccardoM) [reply phase-6/transfer-dtag.md:103]: Yeah I think it's better to tag v0.12.4 and tell people to use that when running this challenges

Comment (RiccardoM) [reply phase-6/transfer-dtag.md:101]: I've added to the Phase 6 README file that all the challenges must be done using Desmos v0.12.3 which we will tag shortly

**Review by kwunyeung — APPROVED** [review]

**Pull request merged**